pandas/core/sorting.py

```diff
@@ -237,9 +244,21 @@ def nargsort(
     items, kind: str = "quicksort", ascending: bool = True, na_position: str = "last"
 ):
     """
-    This is intended to be a drop-in replacement for np.argsort which
-    handles NaNs. It adds ascending and na_position parameters.
-    GH #6399, #5231
+    Intended to be a drop-in replacement for np.argsort which handles NaNs.
+
+    Adds ascending and na_position parameters.
+
+    (GH #6399, #5231)
+
+    Parameters
+
+    ----------
+
+    kind : str, default 'quicksort'
+
+    ascending : bool, default True
+
+    na_position : {'first', 'last'}, default 'last'
     """
```

Comment:
all these blank lines should be removed, starting at `Parameters` there shouldn't be any